doc/source/faq.rst

```diff
@@ -0,0 +1,76 @@
+.. _faq:
+
+==========================
+Frequently Asked Questions
+==========================
+
+.. contents:: Contents
+   :local:
+
+
+Inverse Solution
+================
+
+How should I regularize the covariance matrix
+---------------------------------------------
+
+The MNE manual suggests regularizing the noise covariance matrix (see
+:ref:`CBBHEGAB`). In MNE-Python, this is done using the regularization methods
+described in [1, 2, 3, 4]. For this the 'auto' option can be used. With this
+option cross-validation will be used to learn the optimal regularization.
+
+    >>> import mne
+    >>> epochs = mne.read_epochs(epochs_path)
+    >>> cov = mne.compute_covariance(epochs, tmax=0., method='auto')
```

Comment:
Did you check to make sure this rendered okay? I think you might need a `::` in the preceding line for it to know this is a code literal. GitHub also seems to render it differently than the code literal below, which makes me think you do need `::`.